style/style.css

```diff
@@ -0,0 +1,1065 @@
+}
+
+.domaintable tbody{
+    width: 100%;
+    padding: 2%;
+}
+
+.domaintable tr{
+    flex: 1;
+}
+
+.domaintable td{
+    height: 100%;
+    width: 100%;
+    border-collapse: collapse;
+    padding: 2%;
+    color: #d9d9d9;
+    justify-content: center;
+    align-items: center;
+    font-family: LatoR;
+    font-size: 14px;
+    text-overflow: ellipsis;
+    min-width: 80px;
+    white-space: nowrap;
+}
+
+.domaintable .row1 td{
+    border-bottom: 1px solid #d9d9d9;
+    margin-bottom: 0px;
+    padding-bottom:4%;
+}
+.domaintable .row2 td{
+    padding-top:4%;
+    margin-top: 0px;
+}
+
+.domaintable td:not(:last-child){
+    border-right: 1px solid #d9d9d9;
+}
+
+.domaintable .flex{
+    flex: 1;
+}
+
+/* package */
+
+.Packages{
+    height: 765px;
+    background: url(../img/Packages_BG.jpg) no-repeat center top / contain;
+}
+
+.Packages>.flex{
+    justify-content: center;
+    height:100%;
+}
+
+.packageContainer{
+    justify-content: space-between;
+    align-items: center;
+    width:100%;
+    height: 100%;
+}
+
+.packageContainer>.flex-column{
+    height:530px;
+    width: 20%;
+    padding:28px;
+    border-top:5px;
+    border-style: solid;
+    background-color: #fff;
+    -webkit-box-shadow: 0px 0px 17px -7px rgba(74,74,74,1);
+    -moz-box-shadow: 0px 0px 17px -7px rgba(74,74,74,1);
+    box-shadow: 0px 0px 17px -7px rgba(74,74,74,1);
+    transition: 0.2s linear;
+}
+
+.packageContainer>.flex-column:hover{
+    width: 30%;
+    -webkit-box-shadow: 0px 0px 17px -3px rgba(74,74,74,1);
+    -moz-box-shadow: 0px 0px 17px -3px rgba(74,74,74,1);
+    box-shadow: 0px 0px 17px -3px rgba(74,74,74,1);
+}
+
+.packageContainer .package1{
+    border-color: #f9d423;
+    color: #f9d423;
+}
+.packageContainer .package2{
+    border-color: #7f8c8d;
+    color: #7f8c8d;
+}
+.packageContainer .package3{
+    border-color: #f85f40;
+    color: #f85f40;
+}
+.packageContainer .package4{
+    border-color: #1abc9c;
+    color: #1abc9c;
+}
+
+.packageContainer .sign{
+    flex: 8;
+}
+
+.packageContainer .info{
+    flex: 18;
+    border-top: 1px solid #b4b4b4;
+    border-bottom: 1px solid #b4b4b4;
+    padding: 8% 0px;
+}
+
+.packageContainer .payment{
+    flex: 7;
+}
+
+.info>table{
+    width:100%;
+    height:100%;
+}
+
+.info td:first-child{
+    width: 84%;
+    font-family: LatoL;
+    font-size: 16px;
+    color: #b4b4b4;
+}
+
+.info td:last-child{
+    width:16%;
+    font-family: LatoS;
+    font-size: 16px;
+    text-align: right;
+}
+
+.payment{
+    align-items: center;
+    justify-content: center;
+    color: #2c2c2c;
+}
+.payment sup{
+    font-family: LatoH;
+    font-size: 17px;
+}
+.payment span{
+    font-family: LatoM;
+    font-weight: bold;
+    font-size: 28px;
+}
+
+.payment sub{
+    font-family: LatoR;
+    font-size: 13px;
+    color: #8d8d8d;
+}
+
+.payment a{
+    text-decoration: none;
+    color: #2c2c2c;
+    font-family: LatoM;
+    font-weight: bold;
+    font-size: 16px;
+}
+
+.payment div:first-child{
```

Comment:
Please avoid such implicit assignment. Its better to add some specific class name to target to improve readability of your code.